### PR TITLE
Fix e2e docker:ssh command to work in any repo

### PR DESCRIPTION
**Pull request by rrennick**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Dependent on #27113 being merged as this uses features added to that PR. 

The current `docker:ssh` script in the `e2e-environment` only works in WC core. The `e2e-environment` names containers using the `basename()` of the repository root (eg WC Admin would be `woocommerce-admin_wordpress-www`). The current script is hardcoded to `woocommerce_wordpress-www`.

Closes # .

### How to test the changes in this Pull Request:

1. `npm run docker:up`
2. ` npm explore @woocommerce/e2e-environment -- npm run docker:ssh`
3. `cat wp-config.php`
4. You should see the WP config file for the test environment.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Fix e2e docker:ssh command to work in any repo.

### CODE REVIEW

File: tests/e2e/env/utils/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
-const getAppRoot = require( './app-root' );
+const { getAppRoot, getAppName } = require( './app-root' );
 const getTestConfig = require( './test-config' );
 
 module.exports = {
     getAppRoot,
+	getAppName,
```
Comment (rodrigoprimo) [inline tests/e2e/env/utils/index.js:6]: This line is indented using tabs and the others are using spaces. I'm not sure which one is the correct option.

**Review by rodrigoprimo — COMMENTED** [review]
I followed the test instructions and could confirm that the `npm run docker:ssh` command is working and the `wp-config.php` file present in the environment used to run the tests. I left an inline comment mentioning a nitpick.

Besides that, it is not clear to me why `package-lock.json` is modified in this PR, especially considering the huge number of changes, but that could be because I'm not familiar with how the new e2e tests environment work and thus I'm missing the connection. Could you please clarify?

File: tests/e2e/env/utils/index.js (hunk 1 above)
Comment (rrennick) [reply tests/e2e/env/utils/index.js:6]: > Could you please clarify?

@rodrigoprimo I had to update package-lock.json to get Travis to run. Indentation fixed in `7c509d8`.

**Review by rodrigoprimo — APPROVED** [review]

**Review by ObliviousHarmony — APPROVED** [review]
LGTM!

**Pull request merged**